bundle/manifests/operator.shipwright.io_shipwrightbuilds.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     listKind: ShipwrightBuildList
     plural: shipwrightbuilds
     singular: shipwrightbuild
-  scope: Namespaced
+  scope: Cluster
```

Comment:
IIRC openshift-pipelines exposes the option when you install the the operator from the console of being cluster wide or per namespace.

If I'm correct, I wonder if we should follow that pattern.

And if so, figure out exactly how they do it.

I'm fine with putting a TODO comment and opening a follow up feature if you like @adambkaplan

Reply:
@gabemontero OLM operators initially allow operators to be namespaced, in which case the operator would only watch one namespace. This practice has been discouraged, and most operators are written to be singletons on a cluster.

Tekton's operator uses the `TektonConfig` object for configuration, which is cluster-scoped. For us to depend on Tekton for installation, we should do the same.

See https://github.com/tektoncd/operator/blob/main/pkg/apis/operator/v1alpha1/tektonconfig_types.go#L33-L40